### ISSUE TITLE
Multiple email addresses are not counted correctly.

### ISSUE THREAD

**Issue by vandebo** [aprotim/cordially#3]
Enter multiple email addresses for an invitee.
Send email.
"1 email" is reported as sent.

**Comment by aprotim** [comment]
I think this is WAI, as only one email is sent, but with multiple recipients.

**Comment by vandebo** [comment]
Indeed, technically one email. Might be more interesting to report the number of recipients.
